TruckMuncher.xcodeproj/project.pbxproj

```diff
@@ -954,7 +959,7 @@
 			baseConfigurationReference = C55C42C1CDA48C09ADBED963 /* Pods.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Developer: Josh Ault (YKW8RZF6K4)";
```

Comment:
:anger: Why you change code signing identity? :anger: Some people just like to watch the world burn :fire: